helm/charts/hpe-csi-driver/README.md

```diff
@@ -96,6 +98,8 @@ helm uninstall hpe-csi --namespace kube-system
 helm delete hpe-csi --purge
 ```
 
+> Note: CRDs are not deleted as part of Helm uninstall.
+
```

Comment:
**Note:** Due to a limitation in Helm, CRDs are not deleted as part of the chart uninstall.

Reply:
Done